web/modules/weather_blocks/src/Plugin/Block/WFOPromoBlock.php

```diff
@@ -28,39 +28,10 @@ public function build()
                 "wfo_promo",
             );
 
-            // If we actually have a story, now we can go about pulling data
-            // from it to pass along to the template.
+            // If we actually have a story, pass its ID on for the template
             if ($promo) {
-                // And return the stuff that's available to our block template.
-                $shortIntro = $promo->get("body")->getValue();
-                $shortIntro = array_pop($shortIntro);
-                $shortIntro = $shortIntro["value"];
-
-                $wfoName = $promo->get("title")->getString();
-                $phone = $promo->get("field_phone_number")->getString();
-
-                $social = [];
-                $facebook = $promo->get("field_facebook_url")->getString();
-                if ($facebook) {
-                    $social["facebook"] = $facebook;
-                }
-
-                $twitter = $promo->get("field_twitter_url")->getString();
-                if ($twitter) {
-                    $social["twitter"] = $twitter;
-                }
-
-                $youtube = $promo->get("field_youtube_url")->getString();
-                if ($youtube) {
-                    $social["youtube"] = $youtube;
-                }
-
                 return [
-                    "name" => $wfoName,
-                    "code" => $grid->wfo,
-                    "shortIntro" => $shortIntro,
-                    "phone" => $phone,
-                    "social" => count($social) ? $social : false,
+                    "node" => $promo->id(),
```

Comment:
Instead of pulling all that content out ourselves, just pass the node ID into the template. We'll use a Twig tweak helper to render the node.